keps/sig-network/563-dual-stack/README.md

```diff
@@ -1386,9 +1390,10 @@ This capability will move to stable when the following criteria have been met.
 
 
 * **How can this feature be enabled / disabled in a live cluster?**
-  - [X] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: IPv6DualStack
-    - Components depending on the feature gate: kube-apiserver, kube-controller-manager, kube-proxy, and kubelet
+  - With this feature moving to stable, the feature will always be enabled.
+  - While disabling the feature is not possible, using it is not required. Any
+    cluster can be provisioned with the `spec.ipFamilyPolicy` field set to
```

Comment:
The `spec.ipFamilyPolicy` applies to services that are created in the cluster. As for the cluster provisioning, the `--cluster-cidr` and `--service-cluster-ip-range` are what determines if the cluster should provide the dual-stack functionalities